### PR TITLE
Expand readme, add possibility to generate inlcude files, Fix division by zero and typo

### PR DESCRIPTION
**Pull request by bartsch-dev**
When a language has no translated pages (in recent events `it`) my script throws a `dvision by zero`-exception.
That and a typo is fixed by this PR.

### CODE REVIEW

**Comment by bartsch-dev** [comment]
From now on the include files for each language are generated too.
Makes it easier to translate them too.

File: _scripts/README.md [hunk 1]
```diff
@@ -1,11 +1,35 @@
-## Add a new langauge
+## Script overview
+
+The Script has following commands available (each can be shown when called with the help parameter `-h` on any of the positional parameters)
+- `$ python _scrtips/automate.py status [-e|--extended] [-m --markdown]` d
```
Comment (koppor) [inline _scripts/README.md]: Is the ` d` at the end important?

Comment (koppor) [inline _scripts/README.md]: Is the ` d` at the end important?

Comment (bartsch-dev) [inline _scripts/README.md]: Nope, it's a typo.

**Pull request merged**